README.md

```diff
@@ -53,7 +54,7 @@ workflows.
 This tool is on PyPI. To install, please run:
 
 ```bash
-$ pip install fixml
+pip install fixml
 ```
 
```

Comment:
```suggestion
export OPENAI_API_KEY=...
```

Reply:
This will be addressed in upcoming PR for #205.